docs/libs/ml/quickstart.md

```diff
@@ -24,4 +24,198 @@ under the License.
+  .setIterations(100)
+  .setRegularization(0.001)
+  .setStepsize(0.1)
+  .setSeed(42)
+
+svm.fit(adultTrain)
+
+{% endhighlight %}
+
+Let's now make predictions on the test set and see how well we do in terms of absolute error
+We will also create a function that thresholds the predictions to the {-1, 1} scale that the
+dataset uses.
+
+{% highlight scala %}
+
+def thresholdPredictions(predictions: DataSet[(Double, Double)])
+: DataSet[(Double, Double)] = {
+  predictions.map {
+    truthPrediction =>
+      val truth = truthPrediction._1
+      val prediction = truthPrediction._2
+      val thresholdedPrediction = if (prediction > 0.0) 1.0 else -1.0
+      (truth, thresholdedPrediction)
+  }
+}
+
+val predictionPairs = thresholdPredictions(svm.predict(adjustedTest))
+
+val absoluteErrorSum = predictionPairs.collect().map{
+  case (truth, prediction) => Math.abs(truth - prediction)}.sum
+
+println(s"Absolute error: $absoluteErrorSum")
+
+{% endhighlight %}
+
+Next we will see if we can improve the performance by pre-processing our data.
+
+## Data pre-processing and pipelines
+
+A pre-processing step that is often encouraged when using SVM classification is scaling
+the input features to the [0, 1] range, in order to avoid features with extreme values dominating the rest.
+FlinkML has a number of `Transformers` such as `StandardScaler` that are used to pre-process data, and a key feature is the ability to
+chain `Transformers` and `Predictors` together. This allows us to run the same pipeline of transformations and make predictions
+on the train and test data in a straight-forward and type-safe manner. You can read more on the pipeline system of FlinkML,
+[here](pipelines.html).
+
+Let first create a scaling transformer for the features in our dataset, and chain it to a new SVM classifier.
+
+{% highlight scala %}
+
+import org.apache.flink.ml.preprocessing.StandardScaler
+
+val scaler = StandardScaler()
+scaler.fit(adultTrain)
+
+val scaledSVM = scaler.chainPredictor(svm)
+
+{% endhighlight %}
+
+We can now use our newly created pipeline to make predictions on the test set. 
+First we call fit again, to train the scaler and the SVM classifier.
+The data of the test set will then be automatically scaled before being passed on to the SVM to 
+make predictions.
+
+{% highlight scala %}
+
+scaledSVM.fit(adultTrain)
+
+val predictionPairsScaled= thresholdPredictions(scaledSVM.predict(predictionsScaled))
+
+val absoluteErrorSumScaled = predictionPairs.collect().map{
+  case (truth, prediction) => Math.abs(truth - prediction)}.sum
+
+println(s"Absolute error with scaled features: $absoluteErrorSumScaled")
+
+{% endhighlight %}
+
+The effect that the transformation has on the rror for this dataset is a bit unpredictable.
+In reality the scaling transformation does
+not fit the dataset we are using, since the features are translated categorical features and as
+such, operations like normalization and standard scaling do not make much sense.
```

Comment:
My thoughts were on using the astroparticle dataset from the "Practical guide to SVM classification". If we merge the MinMaxScaler soon I can use that one.